zproject/backends.py

```diff
@@ -733,6 +750,10 @@ def sync_avatar_from_ldap(self, user: UserProfile, ldap_user: _LDAPUser) -> None
 
             ldap_avatar = ldap_user.attrs[avatar_attr_name][0]
 
+            if type(ldap_avatar) != bytes:
+                logging.warn("Skipping avatar sync for %s, wrong data type: %s", user.email, str(type(ldap_avatar)) )
+                return
+
```

Comment:
I'm confused by this, it seems unrelated to the content of the PR. Is this a fix for some kind of bug/error you've encountered with the avatar sync? If so, could you submit it as a separate PR and remove the change from this one?

Reply:
I moved it to a follow up branch